src/analysis/special_functions/exp_log.lean

```diff
@@ -753,6 +753,20 @@ begin
     { exact neg_zero.symm } },
 end
 
+/-- The function `x * log (1 + t / x)` tends to `t` at `+∞`. -/
+lemma tendsto_mul_log_one_plus_div_at_top (t : ℝ) :
+  tendsto (λ x, x * log (1 + t / x)) at_top (𝓝 t) :=
+begin
+  have h₁ : tendsto (λ h, h⁻¹ * log (1 + t * h)) (𝓝[{0}ᶜ] 0) (𝓝 t),
```

Comment:
Is this worth having as a standalone lemma?

Reply:
This is a direct consequence of the formula for the derivative of `λ h, log (1 + t * h)`.  I don't think we should elevate the derivatives of random expressions into lemmas